src/requests/team.ts

```diff
@@ -271,34 +273,55 @@ async function getBySlug(client: GraphQLClient, slug: string) {
   return response.teams[0];
 }
 
-interface GetBopsSubmissionURL {
+interface GetEncryptedIntegrations {
   teams: {
     integrations: {
       bopsSubmissionURL: string | null;
+      bopsSecret: string | null;
     } | null;
   }[];
 }
 
-async function getBopsSubmissionURL(
-  client: GraphQLClient,
-  slug: string,
-  env: PlanXEnv,
-) {
+interface Integrations {
+  bopsSubmissionURL?: string;
+  bopsToken?: string;
```

Comment:
Yep - good call - explicit is better!

The naming convention I'm aiming for with columns is `*_secret` and then in this `DecryptedIntegrations` interface we can use the "real" name (this might be GovPay API key, Uniform client, Bops Token etc)